package.json

```diff
@@ -158,11 +158,16 @@
     ]
   },
   "dependencies": {
+    "@date-io/date-fns": "1.x",
     "@draftbit/react-theme-provider": "^2.1.1",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/pickers": "^3.2.10",
     "color": "^3.1.2",
+    "date-fns": "^2.16.1",
     "dateformat": "^3.0.3",
     "expo-font": "^8.1.1",
     "expo-step-indicator": "0.1.2",
+    "react-dom": "^16.13.1",
```

Comment:
i think this has to live under `devDependencies`, let me get back to you in a bit